Functions/displayUserDetails.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:io';
+import 'dart:convert';
+import '../Student.dart';
+void displayData(Map<int,Map<String,dynamic>> data){
+
+  String choice;
+  print("--------------------------------------------------------------");
+  print("------------------------Displaying Data-----------------------");
+  print("--------------------------------------------------------------");
+  stdout.write("Press, 1 for Sorting in Ascending order, 2 for Desending order, cancel to go back to previous menu");
+
+  List<Student> studentData = [];
```

Comment:
variable names can be named more intuitively, reading the variable name should make it clear that what type of variable it is. 
Example: `students` or `studentList` could be more intuitive.